Resources/views/Menu/sonata_menu.html.twig

```diff
@@ -60,6 +60,7 @@
         <a href="#">
             {% set translation_domain = item.attribute('label_catalogue') %}
             {% set icon = item.attribute('icon')|default('') %}
+            {{ icon|raw }}
```

Comment:
According to commit https://github.com/sonata-project/SonataAdminBundle/commit/df42796fde5d850f04440fb344cac25caad25268#diff-7aad62c164c47d66f90ce813f6f91c49R62, I think this should be on `spanElement`.

cc @rande